README.md

```diff
@@ -1 +1,62 @@
-# client-examples
\ No newline at end of file
+# Description of client-examples
+This repository contains example of Kafka clients. 
+Both producer and consumer clients are transformed into docker images which allows to deploy them on OpenShift platform.
+This may serve as basic usage example for [Strimzi][strimziGH]project.
+
+This repository also contains `deployment.yaml` file with full deployment configuration of producer, consumer, and definitions for topic and user operator.
+Logging configuration can be found in the `log4j2.properties` file for producer and consumer separately.
+
+## Build
+After cloning this repository to your folder Hello World example is fully ready to be build with maven.
```

Comment:
You should at least say what the build requirements are (`mvn`, `make`, `docker`).